package.json

```diff
@@ -1,29 +1,36 @@
 {
-	"name": "serialize-error",
+	"name": "@ponbike/serialize-error",
 	"version": "11.0.2",
 	"description": "Serialize/deserialize an error into a plain object",
 	"license": "MIT",
-	"repository": "sindresorhus/serialize-error",
+	"repository": "ponbike/serialize-error",
 	"funding": "https://github.com/sponsors/sindresorhus",
 	"author": {
 		"name": "Sindre Sorhus",
 		"email": "sindresorhus@gmail.com",
 		"url": "https://sindresorhus.com"
 	},
 	"type": "module",
-	"exports": "./index.js",
+	"exports": {
+		"import": "./src/index.js",
+		"require": "./dist/index.cjs",
+		"types": "./src/index.d.ts"
+	},
 	"engines": {
 		"node": ">=14.16"
 	},
 	"scripts": {
 		"//test": "xo && ava && tsd",
```

Comment:
Do we still need this?

Reply:
I also made a PR to the author that he can add the support